forms-flow-web/src/components/ServiceFlow/list/ServiceTaskListView.js

```diff
@@ -97,155 +99,258 @@ const ServiceTaskListView = React.memo((props) => {
+                                  : "rotate(0deg)"
+                              }`,
+                            }}
+                            aria-hidden="true"
+                          />
+                        </div>
+                      </Col>
+                    )}
+                  </Row>
+                  {expandedTasks[task.id] &&
+                    task?._embedded?.variable?.some(
+                      (eachVariable) =>
+                        eachVariable.name !== "applicationId" &&
+                        eachVariable.name !== "formName" &&
+                        selectedTaskVariables[eachVariable.name] === true
+                    ) && (
+                      <>
+                        <hr />
+                        <Row className="p-2">
+                          {taskListAttributes?.map((eachVariable, index) => {
+                            if (
+                              eachVariable.name !== "applicationId" &&
+                              eachVariable.name !== "formName" &&
+                              selectedTaskVariables[eachVariable.name] === true
+                            ) {
+                              const data = taskvariables?.find(
+                                (variableItem) =>
+                                  variableItem.name === eachVariable.name
+                              );
+                              return (
+                                <Col xs={2} key={index}>
+                                  <div
+                                    className="col-12"
+                                    style={{ wordBreak: "break-all" }}
+                                  >
+                                    <h6 className="fw-bold">{data?.label}</h6>
+                                  </div>
+                                  <div className="d-flex col-12">
+                                    <h6>
+                                      <u className="fw-light text-decoration-none ">
+                                        {eachVariable.value}
+                                      </u>
+                                    </h6>
+                                  </div>
+                                </Col>
+                              );
+                            }
+                          })}
+                        </Row>
+                      </>
+                    )}
+                </div>
+              );
+            })}
+
             <div className="d-flex justify-content-between">
               <div className="ms-2">
-                  <span>
+                <span>
                   {t("Rows per page")} :
                   <DropdownButton
                     className="ms-2 d-inline"
                     drop="down"
                     variant="secondary"
-                      title={selectedLimitValue}
+                    title={selectedLimitValue}
+                    style={{ display: "inline" }}
```

Comment:
inline style is already added as  bootstrap class

Reply:
@AbijithS-aot